engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -610,7 +659,13 @@ yield switch (identifier) {
           expressions.remove(expressions.size()-1);
         }
         var list = CollectionConverters.asScala(expressions.iterator()).toList();
-        yield new IR$Expression$Block(list, last, getIdentifiedLocation(body), false, meta(), diag());
+        var loc = getIdentifiedLocation(body, 0, 1);
```

Comment:
can you add some comments on those magic constants?

Reply:
OK, done in [ea1a895](https://github.com/enso-org/enso/pull/3849/commits/ea1a895e583bf706a58e1bce39fd31d2767a8e99)